mysql-test/suite/galera/t/galera_auto_inc_off.test

```diff
@@ -0,0 +1,51 @@
+#
+# 1) Check that the values of auto-increment step and offset,
+#    which are set by the user, are saved and restored when user
+#    turning off the automatic control for these options.
+# 2) Check that these options are successfuly passed to new session.
+# 3) Check that the same logic works for the auto-increment parameters
+#    passed via command line.
+#
+--source include/galera_cluster.inc
+--source include/have_innodb.inc
+
+SET GLOBAL auto_increment_increment=10;
+SET GLOBAL auto_increment_offset=7;
+SET GLOBAL wsrep_auto_increment_control='OFF';
+
+SHOW GLOBAL VARIABLES LIKE '%auto_increment%';
+
+--let $galera_connection_name = node_1a
+--let $galera_server_number = 1
+--source include/galera_connect.inc
+
+--connection node_1a
+
+SHOW VARIABLES LIKE '%auto_increment%';
+
+--connection node_1
+
+SET GLOBAL wsrep_auto_increment_control='ON';
+
```

Comment:
Let me explain my logic. In general, we do not know what the offset received by each of the nodes, and which step will be used when wsrep_auto_increment_control = 'ON'. We can put into the .result file some values, which given by the current algorithm, but we have no guarantee that they will be same tomorrow. We have occasionally saw TC, which failed on the Jenkins because the nodes "node_1" and "node_2" obtained auto_increment_offset = 2 (at the node #1) and auto_increment_offset = 1 (at the node #2), instead of auto_increment_offset = 1 (at the node #1) and auto_increment_offset = 2 (at the node #2) (this order is recorded in .result file). But if you think it is right to enter values relying on the current version of algorithm and the current order of the start of nodes (node #1 always first), then I can add it.